openedx/core/djangoapps/lang_pref/views.py

```diff
@@ -7,23 +7,21 @@
 
 from django.conf import settings
 from django.http import HttpResponse
-from django.utils.translation import LANGUAGE_SESSION_KEY
 from django.views.decorators.csrf import ensure_csrf_cookie
 
 from openedx.core.djangoapps.lang_pref import LANGUAGE_KEY
-from openedx.core.djangoapps.lang_pref.helpers import set_language_cookie
+from openedx.core.djangoapps.lang_pref.helpers import get_language_cookie, set_language_cookie
 
 
 @ensure_csrf_cookie
-def update_session_language(request):
+def update_cookie_language(request):
```

Comment:
Should we go to a more generic name like `update_language` since it was doing both, and the implementation should be able to change?

Reply:
Yes, done.